pallets/accounts/src/lib.rs

```diff
@@ -881,16 +950,16 @@ pub mod pallet {
 				let amount = data[1].clone().into_uint().ok_or_else(|| "Invalid data")?; // Amount of FRAG token locked/unlocked (`data[1]`)
 
 				// Lock period (`data[2]`). In case of Unlock event, it is zero.
-				let locktime = data[2].clone().into_uint().unwrap_or(U256::from(0));
+				let lock_period = data[2].clone().into_uint().unwrap_or(U256::from(999));
```

Comment:
WHat is 999?

Reply:
Since now our smart contract on Ethereum returns the integer representing the enum of the lock periods (`0` = 2 weeks, `1` = 1 month, etc...), I am now setting a default value of `999` in case of Unlock event, where nothing is returned, but it is still needed later to populate `EthLockUpdate` in `send_unsigned_transaction`.

This `999` value will not be taken into account in the `internal_lock_event` function where there are all the calculations for internal Frag, Tickets and NOVA.